openid-caep-interoperability-profile-1_0.md

```diff
@@ -0,0 +1,204 @@
+
+### Verification Endpoint {#verification-endpoint}
+The Transmitter Configuration Metadata MUST include the `verification_endpoint` field. The specified endpoint MUST provide a way to request verification events to be sent.
+
+### Authorization Schemes
+The Transmitter Configuration Metadata MUST include the `authorization_schemes` field and its value MUST include the value
+
+~~~json
+{
+    "spec_urn": "urn:ietf:rfc:6749"
+}
+~~~
+
+### Streams {#common-stream-configuration}
+In all streams created by the Transmitter, the following MUST be true:
+
+#### Delivery {#common-delivery}
+A Transmitter MUST be able to accept a Create Stream request that includes either of the following delivery methods:
+
+* urn:ietf:rfc:8935 (Push)
+* urn:ietf:rfc:8936 (Poll)
+
+The `delivery` field MUST be present in the Configuration of any Stream generated by the Transmitter, and its value MUST include one of the two delivery methods listed above.
+
+#### Stream Control
+The following Stream Configuration API Methods MUST be supported:
+
+**Creating a Stream**
+: Receivers MUST be able to create a Stream with the Transmitter using valid authorization with the Transmitter. The Transmitter MAY support multiple streams with the same Receiver
+
+**Reading Stream Configuration**
+: A Receiver MUST be able to obtain current Stream configuration from the Transmitter by providing a valid authorization
+
+**Getting the Stream Status**
+: A Receiver MUST be able to obtain the current Stream status from the Transmitter by providing a valid authorization
+
+**Stream Verification**
+: A Receiver MUST be able to verify the liveness of the Stream by requesting that the Transmitter send it a Stream Verificaiton event by providing a valid authorization
+
+## Receivers {#common-receivers}
+Receivers MUST implement the following features:
+
+### Delivery Methods {#common-receiver-delivery}
+Receivers MUST be able to accept events using the Push-Based Security Event Token (SET) Delivery Using HTTP {{RFC8935}} specification and the Poll-Based Security Event Token (SET) Delivery Using HTTP {{RFC8936}} specification.
+
+### Implicitly Added Subjects {#common-receiver-subjects}
+Receivers MUST assume that all subjects are implicitly included in a Stream, without any `AddSubject` method invocations.
+
+## Event Subjects {#common-event-subjects}
+The following subject identifier formats from "Subject Identifiers for Security Event Tokens" {{RFC9493}} MUST be supported:
+
+* `email`
+* `iss_sub`
+
+Receivers MUST be prepared to accept events with any of the subject identifier formats specified in this section. Transmitters MUST be able to send events with at least one of subject identifier formats specified in this section.
+
+## Event Signatures
+All events MUST be signed using the `RS256` algorithm using a minimum of 2048-bit keys.
```

Comment:
I missed this in my first pass. What would you think of using `ES256` instead of (or in addition to) `RS256`? I recognize that `RS256` is often the default implementation, but `ES256` is more secure and generally preferred over `RS256`.